### PR TITLE
Write Loop Lab

### PR DESCRIPTION
**Pull request by kcleland0818**

### CODE REVIEW

**Review by linda-seiter — COMMENTED** [review]
Could the while loop and do while have them implement something that is not a counting loop to show why you would use it instead of a for loop?  reading in or generating random sales until a minimum sales quota is reached, or something like that?

**Comment by kcleland0818** [comment]
> Could the while loop and do while have them implement something that is not a counting loop to show why you would use it instead of a for loop? reading in or generating random sales until a minimum sales quota is reached, or something like that?

Let me take this lab and turn it into a lesson instead. I think it is important to see that you _can_ solve a problem or use any of these loops. But why you _should_ use a for loop in this case over a while or do-while.

I'll rewrite this lab to have some problems that are more appropriate for the given loops.

File: README.md [hunk 1]
```diff
@@ -2,30 +2,137 @@
...
+5 years have passed
+6 years have passed
+7 years have passed
+8 years have passed
+9 years have passed
+10 years have passed
+```
+
+- Make sure the case that outputs "0 year(s) have passed" doesn't run anymore.
+  - Hint: It has to do with he way `currentYear` is initialized.
```
Comment (developerjaym) [inline README.md]: Missing the 't' in 'the'

Comment (kcleland0818) [reply README.md]: Thanks for catching this!

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]
looks good!

**Pull request merged**